db/seed.sql

```diff
@@ -7667,6 +7669,10 @@ LOCK TABLES `eav_attribute_option_value` WRITE;
 INSERT INTO `eav_attribute_option_value` VALUES (1,1,0,'Male');
 INSERT INTO `eav_attribute_option_value` VALUES (2,2,0,'Female');
 INSERT INTO `eav_attribute_option_value` VALUES (3,3,0,'Not Specified');
+INSERT INTO `eav_attribute_option_value` VALUES (4,4,1,'Paperback');
```

Comment:
One has `store_id` of `0` (which I think means that it's the global default) and the other has `store_id` of `1` (which is the only actual store we have).  Not sure if or why it needs both, but I did it through the UI and this is what it put in the database.  It's possibly a side-effect of being in "single store mode", which simplifies the UI a little by hiding store selectors in various places (since there's only one store).